design-documents/asynchronous-import/base-extension.md

```diff
@@ -23,16 +26,16 @@ Path is relative from Magento Root folder
 
 ```
 {
-  "source": {
-      "import_data": "var/catalog_product.csv",
-      "import_type": "local_path",
-      "uuid": "UUID",
-      "format": {
-          "csv_separator": "string",
-          "csv_enclosure": "string",
-          "csv_delimiter": "string",
-          "multiple_value_separator": "string"
-      }
+  "uuid": "UUID",
```

Comment:
will this UUID be returned by the system/api itself rather than the user? if that is true, then we can remove it from the Request, and also we can name it just an ID, rather than UUID. Since UUID is one of the strategy to generate the IDs, which can be changed in the future. We should not name the underlying strategy of the ID generation, for the name of the Entity ID.

Reply:
UUID can be defined by the client, but at the same time IF its not set, we are generating it on Magento side, thats why this param is here

Reply:
I understand your point related to IDs naming.
Its correct idea if we are talking about system that have this pattern everywhere. 
Unfortunatelly its not in Magento, so _id is everywhere in the system are Database Ids. 
Same time you are right that user should not be aware of way how ID is generated. 

In our implementation its explicitly UUID, cause its currently on our code level.
May I ask you that we leave it as it is for now, and will have another discussion.

Btw. you can raise this topic on next Architectural meeting so there will be a lot of people to discuss and this may define pattern for future Magento standard of IDs naming

Reply:
ok I will raise this within architecture meeting